ecl/regress/quantile_e1.ecl

```diff
@@ -0,0 +1,44 @@
+/*##############################################################################
+
+    HPCC SYSTEMS software Copyright (C) 2015 HPCC Systems.
```

Comment:
Missing registered trademark symbol on HPCC Systems - similarly for all other newly added tests etc.